exonum/src/api/private/system.rs

```diff
@@ -90,6 +90,16 @@ struct PeersInfo {
     outgoing_connections: HashMap<SocketAddr, IncomingConnection>,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+struct EnabledInfo {
+    pub enabled: bool,
```

Comment:
These fields don't need to be public, do they? Additionally, I'd move definitions into functions where they are used, but I don't insist on that.